Gruntfile.js

```diff
@@ -89,6 +89,7 @@ module.exports = function(grunt) {
 		connect: {
 			server: {
 				options: {
+					hostname: '0.0.0.0',
```

Comment:
I set it explicitly because I wasn't aware of that ticket :-)

I only needed it to test locally, feel free to remove it!